oaknational/Oak-Web-Application
# fix: add aria-hidden to presentational images

## Description
- Add `aria-hidden` to `CMSImage`s where `isPresentational` is truthy
- Explicitly hide blog post author images

## Issue(s)

Fixes #

## How to test

Go to [the preview deploy](https://deploy-preview-782--oak-web-application.netlify.thenational.academy/)
- All blog thumbnails should be `aria-hidden`/`alt=""`
- Any images marked as `isPresentational` in the CMS should be `aria-hidden`/`alt=""`

## Checklist
